apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowFilters.cs

```diff
@@ -0,0 +1,407 @@
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells
+        /// from rows whose keys satisfy the given RE2 regex. In other words, passes
+        /// through the entire row when the key matches, and otherwise produces an
+        /// empty row.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that, since row keys can contain arbitrary bytes, the `\C` escape
+        /// sequence must be used if a true wildcard is desired. The `.` character
+        /// will not match the new line character `\n`, which may be present in a
+        /// binary key.
+        /// </para>
+        /// </remarks>
+        /// <param name="regex">
+        /// The RE2 regex used to match row keys.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter RowKeyRegex(RowKey regex)
+        {
+            throw new InvalidOperationException();
+        }
+
+        // TODO: Seems like this is 0-1. Verify and document it.
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches all cells
+        /// from a row with probability p, and matches no cells from the row with
+        /// probability 1-p.
+        /// </summary>
+        /// <param name="probability">
+        /// The probability with which rows should be matched.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter RowSample(double probability)
+        {
+            throw new InvalidOperationException();
+        }
+
+        // TODO: Copy docs from RowFilter.Sink, but only once the formatting issues are resolved.
+        /// <summary>
+        /// ADVANCED USE ONLY. Creates a new <see cref="RowFilter"/> instance which is a
+        /// hook for introspection into the RowFilter. Outputs all cells directly to the
+        /// output of the read rather than to any parent filter.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// </para>
+        /// As with an <see cref="Interleave"/>, duplicate cells are possible, and appear
+        /// in an unspecified mutual order. In this case we have a duplicate with column
+        /// "A:B" and timestamp 2, because one copy passed through the all filter while
+        /// the other was passed through the label and sink. Note that one copy has label
+        /// "foo", while the other does not.
+        /// <para>
+        /// Cannot be used within the predicateFilter, trueFilter, or falseFilter of a
+        /// <see cref="Condition"/>.
+        /// </para>
+        /// </remarks>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter Sink()
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which replaces each cell's
+        /// value with the empty string.
```

Comment:
I think that it would be useful to add that this filter is useful when the objective is to perform counts rather than get the values.

Reply:
Done